### PR TITLE
AI chat app refresh

### PR DESCRIPTION
**Pull request by diberry**
Remove unnecessary prereq
Move to GitHub stable markdown for devcontainer and Codespaces.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/ai/get-started-app-chat-scaling-with-azure-container-apps.md](https://github.com/dotnet/docs/blob/f6cb49fe886d2674b515a969eb1546719997377c/docs/ai/get-started-app-chat-scaling-with-azure-container-apps.md) | [CustomerIntent: As a .NET developer new to Azure OpenAI, I want to scale my Azure OpenAI capacity to avoid rate limit errors with Azure Container Apps.](https://review.learn.microsoft.com/en-us/dotnet/ai/get-started-app-chat-scaling-with-azure-container-apps?branch=pr-en-us-40976) |
| [docs/ai/get-started-app-chat-template.md](https://github.com/dotnet/docs/blob/f6cb49fe886d2674b515a969eb1546719997377c/docs/ai/get-started-app-chat-template.md) | [CustomerIntent: As a .NET developer new to Azure OpenAI, I want deploy and use sample code to interact with app infused with my own business data so that learn from the sample code.](https://review.learn.microsoft.com/en-us/dotnet/ai/get-started-app-chat-template?branch=pr-en-us-40976) |

### CODE REVIEW

**Comment by diberry** [comment]
#assign-reviewer: alexwolfmsft, mcleanbyron

**Review by alexwolfmsft — APPROVED** [review]

File: docs/ai/get-started-app-chat-scaling-with-azure-container-apps.md [hunk 1]
```diff
@@ -26,9 +26,9 @@ ms.custom: devx-track-dotnet, devx-track-dotnet-ai
 
 #### [Visual Studio Code](#tab/visual-studio-code)
 
-* [Azure Developer CLI](/azure/developer/azure-developer-cli/install-azd)
-* [Docker Desktop](https://www.docker.com/products/docker-desktop/) - start Docker Desktop if it's not already running
-* [Visual Studio Code](https://code.visualstudio.com/) with [Dev Container Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
+1. [Docker Desktop](https://www.docker.com/products/docker-desktop/) - start Docker Desktop if it's not already running
```
Comment (gewarren) [inline docs/ai/get-started-app-chat-scaling-with-azure-container-apps.md:29]: This shouldn't be an ordered list, should it?

File: docs/ai/get-started-app-chat-template.md [hunk 2]
```diff
@@ -68,7 +68,6 @@ To follow along with this article, you need the following prerequisites:
 1. Azure account permissions - Your Azure Account must have Microsoft.Authorization/roleAssignments/write permissions, such as [User Access Administrator](/azure/role-based-access-control/built-in-roles#user-access-administrator) or [Owner](/azure/role-based-access-control/built-in-roles#owner).
```
Comment (gewarren) [inline docs/ai/get-started-app-chat-template.md:68]: Same comment here and in the Codespaces prereqs.

**Comment by diberry** [comment]
@gewarren Can you review once more and merge if you approve?

**Review by gewarren — APPROVED** [review]

**Pull request merged**